game_state.py

```diff
@@ -13,11 +13,10 @@
 _game_map = {}  # game_id -> set(websockets)
 
 
-GAME_STATES = (
-    'WRITE',
-    'ASSIGN',
-    'READ',
-)
+class GameStates(Enum):
```

Comment:
out of curiosity, what's the added benefit for converting the dictionary to an enum?